examples/text.rs

```diff
@@ -121,8 +156,23 @@ impl nmg::Update for Demo {
                 1.0 * angle.cos(),
             ) + alg::Vec3::fwd() * 1.0,
         );
+
+        // Rotate text
         components.transforms.set_orientation(
             self.text_1.unwrap(),
+            alg::Quat::axis_angle_raw(alg::Vec3::fwd(), angle),
+        );
+
+        // Rotate label
+        components.transforms.set_orientation(
+            self.label_1.unwrap(),
+            alg::Quat::axis_angle_raw(alg::Vec3::up(), -angle)
```

Comment:
To be honest, I'd prefer it if we only keep the second rotation (line 170) for the example.